MAVProxy/modules/mavproxy_gopro.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+'''arm/disarm command handling'''
```

Comment:
hi Arthur, can you update the comments in the top of the module to explain what it does and how you need to setup the gopro to use it?